tests/test_scripts.py

```diff
@@ -558,15 +558,15 @@ def process_lane(self, lane):
         worklist = script.considered.pop(0)
         eq_(self._default_library, worklist.get_library(self._db))
         eq_(self._default_library.name, worklist.display_name)
-        eq_([good, bad], worklist.children)
+        eq_(set([good, bad]), set(worklist.children))
```

Comment:
This branch changed the order of the children returned by `Lane.children`. My best guess is that something happened in the internals because I added a field to the `lanes` table.